src/components/Capture/Document.js

```diff
@@ -44,15 +45,31 @@ class Document extends Component {
     nextStep()
   }
 
-  handleUpload = (blob) =>
-    this.handleCapture({ blob, sdkMetadata: { captureMethod: 'html5' } })
+  handleUpload = (blob, isResizedImage = false) => {
+    this.handleCapture({
+      blob,
+      sdkMetadata: { captureMethod: 'html5', isResizedImage },
+    })
+  }
 
   handleError = () => this.props.actions.deleteCapture()
 
+  handleFileSelected = (file) => {
+    let isResizedImage = false
+    const error = validateFileTypeAndSize(file)
+    if (error === 'INVALID_SIZE' && file.type.match(/image.*/)) {
```

Comment:
I'm not sure about practices in this project but I'd suggest using const/enum here instead of string directly